multi/imxrt-multi/cm4.c

```diff
@@ -207,6 +245,10 @@ static ssize_t chanRead(id_t id, void *buff, size_t bufflen)
 		}
 	}
 
+	if (i == 0 && m4_common.channel[id].state.nonblock == 1) {
+		i = -EWOULDBLOCK;
+	}
```

Comment:
I'm not sure about this, as it's always non-blocking, but if we open with `O_NONBLOCK` then we'll return `EWOULDBLOCK`, otherwise zero anyway... Maybe for now always assume `O_NONBLOCK`? But I guess as a hot-fix you need it's ok.

Reply:
Blocking mode is still used by `cm4-tool` so we can't always return `EWOULDBLOCK`.